src/analyses/call_graph.cpp

```diff
@@ -42,7 +45,12 @@ void call_grapht::add(
     {
       const exprt &function_expr=to_code_function_call(i_it->code).function();
       if(function_expr.id()==ID_symbol)
-        add(function, to_symbol_expr(function_expr).get_identifier());
+      {
+        const irep_idt &callee=to_symbol_expr(function_expr).get_identifier();
+        add(function, callee);
+        if(collect_callsites)
+          callsites[{function, callee}].insert(i_it);
```

Comment:
Why does this not use the newly introduced `add(caller, callee, callsite)` function?

Reply:
It does -- fixed in a later commit